README.md

```diff
@@ -96,6 +97,12 @@ Building need be done only once if no source code is changed.
     cmake .. -DCMAKE_INSTALL_PREFIX=..
     make -j 4 2>&1 | tee build.log
 ```
```

Comment:
This does not include all the information needed to build GSI.

Reply:
updated with more details.